docs/transactions.md

```diff
@@ -1,8 +1,10 @@
 # Transactions
 
-* [Creating and using transactions](#creating-and-using-transactions)
-* [Transaction decorators](#transaction-decorators)
-* [Using `QueryRunner` to create and control state of single database connection](#using-queryrunner-to-create-and-control-state-of-single-database-connection)
+- [Transactions](#transactions)
```

Comment:
I think we don't need this top-level category since there is a single item only?